mzcc.h

```diff
@@ -2,25 +2,24 @@
 #define MAZUCC_H
 
 #include <stdbool.h>
+#include <stdint.h>
 #include "dict.h"
 #include "list.h"
 #include "util.h"
 
-enum {
+enum TokenType {
     TTYPE_IDENT,
     TTYPE_PUNCT,
     TTYPE_NUMBER,
     TTYPE_CHAR,
     TTYPE_STRING,
+    /* Total number of type, the last element in TokenType */
+    __TTYPE_NUM,
```

Comment:
Since you depend on the value of `__TTYPE_NUM`, identifier TTYPE_IDENT should be set to 0 explicitly.

Reply:
Yes, but this is not used yet, I remove it after rebasing